osu.Game/Overlays/Settings/Sections/Input/JoystickSettings.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Input.Handlers.Joystick;
+using osu.Framework.Localisation;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Localisation;
+
+namespace osu.Game.Overlays.Settings.Sections.Input
+{
+    public class JoystickSettings : SettingsSubsection
+    {
+        private readonly JoystickHandler joystickHandler;
+        protected override LocalisableString Header => JoystickSettingsStrings.JoystickGamepad;
+        private readonly BindableNumber<float> deadzoneThreshold = new BindableNumber<float>();
+        private readonly Bindable<bool> enabled = new BindableBool(true);
+
+        public JoystickSettings(JoystickHandler joystickHandler)
+        {
+            this.joystickHandler = joystickHandler;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Children = new Drawable[]
+            {
+                new SettingsCheckbox
+                {
+                    LabelText = CommonStrings.Enabled,
+                    Current = enabled
+                },
+                new DeadzoneSetting
+                {
+                    LabelText = JoystickSettingsStrings.DeadzoneThreshold,
+                    Current = deadzoneThreshold
+                },
+            };
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            enabled.BindTo(joystickHandler.Enabled);
+            deadzoneThreshold.BindTo(joystickHandler.DeadzoneThreshold);
+            enabled.BindValueChanged(e => deadzoneThreshold.Disabled = !e.NewValue, true);
+        }
+
+        private class DeadzoneSetting : SettingsSlider<float, DeadzoneSlider>
+        {
+            public DeadzoneSetting()
+            {
+                KeyboardStep = 0.005f;
```

Comment:
Not really sure why this step is so tiny. Does anyone really need this fine-grained control over the deadzone? I'd say the step could be 10x what it is right now and nobody should have a problem with that...

Reply:
Osu's default deadzone was `0.075f`, I figured a step of 0.005 would make that values like that easier to get to, though I can consider 0.01 (about 100 choices from 0 to 0.95). A step bigger than that seems way too big of a skip imo.

Reply:
I could live with 0.01 too I guess.